geoh5py/workspace/__init__.py

```diff
@@ -1,24 +0,0 @@
-# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

Comment:
oh you moved workspace to geoh5py

But if somehwere in one of our package we are calling from workspace.workspace import Workspace?

Reply:
Most packages do from `from geoh5py import Workspace` or `from geoh5py.workspace import Workspace`, which is still supported here.

Reply:
But fair enough. I will raise a deprecation warning so we have time to adjust.